### ISSUE TITLE
New version 3.2.5 fail to install on MacOS

### ISSUE THREAD

**Issue by Eric-Guo** [akihikodaki/cld3-ruby#24]
```bash
cat /usr/local/lib/ruby/gems/2.6.0/extensions/x86_64-darwin-18/2.6.0/cld3-3.2.5/gem_make.out
```

```log
current directory: /usr/local/lib/ruby/gems/2.6.0/gems/cld3-3.2.5/ext/cld3
/usr/local/opt/ruby/bin/ruby -I /usr/local/lib/ruby/site_ruby/2.6.0 -r ./siteconf20200103-51270-1o99whl.rb extconf.rb
Could not make proto path relative: feature_extractor.proto: No such file or directory
Could not make proto path relative: sentence.proto: No such file or directory
Could not make proto path relative: task_spec.proto: No such file or directory
creating Makefile

current directory: /usr/local/lib/ruby/gems/2.6.0/gems/cld3-3.2.5/ext/cld3
make "DESTDIR=" clean

current directory: /usr/local/lib/ruby/gems/2.6.0/gems/cld3-3.2.5/ext/cld3
make "DESTDIR="
compiling nnet_language_identifier_c.cc
nnet_language_identifier_c.cc:21:10: fatal error: 'nnet_language_identifier.h' file not found
#include "nnet_language_identifier.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [nnet_language_identifier_c.o] Error 1

make failed, exit code 2
```

**Comment by Eric-Guo** [comment]
3.2.4 works fine and `protobuf` installed by `brew`

**Comment by akihikodaki** [comment]
Fixed with commit d212b057f30587ba991b64f418864dd3868fa82f.

**Comment by Eric-Guo** [comment]
Confirm works right at 3.2.6
